cluster/addons/dns/skydns-rc.yaml.in

```diff
@@ -61,4 +66,26 @@ spec:
         - containerPort: 53
           name: dns-tcp
           protocol: TCP
+        livenessProbe:
+          httpGet:
+            path: /healthz
+            port: 8080
+            scheme: HTTP
+          initialDelaySeconds: 30
+          timeoutSeconds: 5
+      - name: healthz
+        image: gcr.io/google_containers/healthz-server:1.0
+        resources:
```

Comment:
I was planning on 10m. What do you mean by outer shared limits? like a limit range on the ns (that kube-system doesn't have).